Base/res/ladybird/about-pages/settings.html

```diff
@@ -990,6 +1045,56 @@ <h3 id="site-settings-title" class="dialog-title"></h3>
                 });
             });
 
+            // DNS settings
+            const loadDnsSettings = settings => {
+                dnsUpstream.value = settings.mode;
+
+                if (settings.mode === "custom") {
+                    dnsServer.value = settings.server;
+                    dnsPort.value = settings.port;
+                    dnsType.value = settings.type;
+                    customDnsSettings.classList.remove("hidden");
+                } else {
+                    dnsServer.value = "";
+                    dnsType.value = "udp";
+                    dnsPort.value = "53";
+                    customDnsSettings.classList.add("hidden");
+                }
+
+                if (settings.forciblyEnabled) {
+                    dnsForciblyEnabled.classList.remove("hidden");
+                } else {
+                    dnsForciblyEnabled.classList.add("hidden");
+                }
+            };
+
+            dnsUpstream.addEventListener("change", () => {
+                if (dnsUpstream.value === "custom") {
+                    customDnsSettings.classList.remove("hidden");
+                    if (dnsServer.value !== "" && dnsPort.value !== "")
```

Comment:
Can you add braces around this `if`? Just because if I format this file with `prettier`, it will otherwise want to put the if and then statements on one line.